sig-architecture/production-readiness.md

```diff
@@ -80,9 +80,28 @@ To become a reviewer:
  * Read/study previous PRR comments and production readiness responses in existing KEPs.
  * Choose some KEPs requiring PRR and perform a review. Put "shadow prod readiness review"
    in your review comments so that the assigned PRR approver knows your intent.
- * After at least one release cycle, if you have shown good judgement and quality reviews,
-   you can propose yourself as approver by submitting a PRR to add your GitHub
-   handle to the `prod-readiness-approvers` alias in [OWNERS_ALIASES].
+
+### Becoming an approver
+
+After serving as reviewer/shadow for at least one release and showing good judgement and quality reviews,
+you can propose yourself as an approver by submitting a PR to add your GitHub
+handle to the `prod-readiness-approvers` alias in [OWNERS_ALIASES].
+
+When submitting the PR, you should include references to KEPs you reviewed that demonstrated a good variety
+of different situations.
+Here is a good starting point (remember that one PR can cover multiple categories):
+
+* Transitions from new to alpha
+* Transitions from alpha to beta
+* Transitions from beta to GA
+* Must have successfully reviewed at least three enhancements that require coordination between multiple components.
+* Must have successfully reviewed at least three enhancements that require version skew consideration (both HA and component skew):
+  does behavior fail safely and eventually reconcile.
+* Must have successfully reviewed at least three enhancements that are outside your primary domain.
```

Comment:
I'd suggest something like this instead:
```suggestion
* Must have successfully reviewed at enhancements owned by at least four SIGs.
```
Since "primary domain" may be a little hard to pin down but the requirement to have reviewed broadly seems important. Though "owning SIG" may not quite be the right key either.

Reply:
I initially read it as "each KEP is owned by at least 4 SIGs".
I guess you mean that you reviewed KEPs originating from at least 4 SIGs, which makes sense to me.